qa/L0_sequence_batcher/sequence_batcher_test.py

```diff
@@ -2848,6 +2850,85 @@ def test_queue_delay_full_min_util(self):
                         self.cleanup_shm_regions(precreated_shm0_handles)
                         self.cleanup_shm_regions(precreated_shm1_handles)
 
+    def test_request_timeout(self):
```

Comment:
Can we add another test case where one of the request of sequence is sent to the server after one of the request in the sequence should be timed-out?